data/LogicHelpers.json

```diff
@@ -29,11 +29,11 @@
     "Blue_Fire": "'Blue Fire' or Buy_Blue_Fire or Blue_Fire_Drop",
     "Fish": "'Fish' or Buy_Fish or Fish_Drop",
     "Fairy": "'Fairy' or Buy_Fairys_Spirit or Fairy_Drop",
-    "Poe": "'Poe' or Buy_Poe or Poe_Drop",
+    # "Poe": "'Poe' or Buy_Poe or Poe_Drop",
     # "Big_Poe": "'Big Poe'",
 
     "has_bombchus": "Bombchus_5 or Bombchus_10 or Bombchus_20 or (found_bombchus and (Buy_Bombchu_5 or Buy_Bombchu_10 or Buy_Bombchu_20)) or Bombchu_Drop",
-    "found_bombchus": "(abundant_bombchus and ('Bombchus' or Bombchus_5_Limited or Bombchus_10_Limited or Bombchus_20_Limited)) or (not abundant_bombchus and Bomb_Bag)",
+    "found_bombchus": "(bombchus_in_logic and ('Bombchus' or Bombchus_5_Non-Repeatable or Bombchus_10_Non-Repeatable or Bombchus_20_Non-Repeatable)) or (not bombchus_in_logic and Bomb_Bag)",
```

Comment:
Hm, we need to make this say Bombchus_5_Non_Repeatable since `-` isn't a valid character in an identifier. I can handle that and change the regex, unless you want to do it.